design/authenticators/application-identity/application-identity.md

```diff
@@ -0,0 +1,464 @@
+# Application Identity
+
+##### Note
+
+In this doc I talk about authenticating hosts with Conjur. Everything I say
```

Comment:
In this doc, I will cover authenticating hosts with Conjur. Everything about authenticating hosts is also relevant to users. For the scope of this document, `host`, `user` or `role` are interchangeable"